less/admin/PermissionsPage.less

```diff
@@ -3,13 +3,8 @@
   border-radius: @border-radius;
   max-width: calc(~'100% - 60px');
```

Comment:
we need to drop this now that we have the common AdminPage container.

also there is a `padding-left: 0 12px;` somewhere in this file, could you remove that please ?